internal/builders/container/README.md

```diff
@@ -152,6 +153,125 @@ jobs:
       registry-password: ${{ secrets.GITHUB_TOKEN }}
 ```
 
+#### With GCP Artifact Registry
+
+The following is an example of pushing an image to an [Artifact Registry](https://cloud.google.com/artifact-registry) in GCP and generating the provenance for that image. In order for you to run this example, you will need to have a [Workload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation) that enables you to exchange a GitHub token for access within GCP. If you have not yet created one or have not created a provider within your existing federation for GitHub, please review the following resources:
+
+- https://gist.github.com/palewire/12c4b2b974ef735d22da7493cf7f4d37 
```

Comment:
I think the `google-github-actions/auth` docs are  more likely to stay up to date.
```suggestion
- [Setting up Workload Identity Federation](https://github.com/google-github-actions/auth#setting-up-workload-identity-federation)
```